src/Ombi.Core/Engine/V2/CalendarEngine.cs

```diff
@@ -47,7 +47,7 @@ public Task<List<CalendarViewModel>> GetCalendarData()
                         new ExtraParams
                         {
                             Overview = e.Season?.ChildRequest?.ParentRequest?.Overview ?? string.Empty,
-                            ProviderId = e.Season?.ChildRequest?.ParentRequest?.ExternalProviderId ?? 0,
```

Comment:
These changes seem unrelated, and it should be using the `ExternalProviderId`

Reply:
Dev version put back in place